packages/webpack-config/src/webpackConfig.ts

```diff
@@ -608,14 +613,7 @@ function createWebpackConfigForMode(
       symlinks: false,
     },
     resolveLoader: {
-      modules: [
-        // resolve loaders from local node_modules to ensure loaders
-        // are found when this package is linked
-        path.join(__dirname, "..", "node_modules"),
-        // resolve loaders from web-pli root node_modules
-        path.join(__dirname, "..", "..", "..", "node_modules"),
-        "node_modules",
-      ],
+      modules: ["node_modules", ...(settings.modulesDirectories ?? [])],
```

Comment:
❓ So, `moduleDirectories` will now need to be explicitly specified? I did some poking around on stack overflow and found that `import.meta.url` can be used in ESM instead of `__dirname`. Perhaps this would be an alternative to consider.